modules/cluster-autoscaler-cr.adoc

```diff
@@ -54,3 +54,8 @@ spec:
 <13> Specify the period to wait before deleting a node after a node has recently been _deleted_. If you do not specify a value, the default value of `10s` is used.
 <14> Specify the period to wait before deleting a node after a scale down failure occurred. If you do not specify a value, the default value of `3m` is used.
 <15> Specify the period before an unnecessary node is eligible for deletion. If you do not specify a value, the default value of `10m` is used.
+
+[NOTE]
+====
```

Comment:
Suggested edits: 
s/autoscaler will remain/autoscaler remains
s/autoscaler will not correct/autoscaler does not correct